thundra/integrations/botocore.py

```diff
@@ -280,14 +309,14 @@ def getRequestType(self, string):
         return Constants.AWS_SERVICE_REQUEST
 
     def inject_span_info(self, scope, wrapped, instance, args, kwargs, response,
-                 exception):
+                         exception):
 
         operation_name, request_data = args
         self.bucket = request_data['Bucket']
 
         scope.span.domain_name = Constants.DomainNames['STORAGE']
         scope.span.class_name = Constants.ClassNames['S3']
-        scope.span.operation_name = self.getRequestType(operation_name)
+        scope.span.operation_name = self.bucket
```

Comment:
what if there is no `Bucket` in the request? For example, when the request is for list buckets